chainer/links/model/vision/resnet.py

```diff
@@ -221,6 +244,150 @@ def predict(self, images, oversample=True):
         return y
 
 
+class ResNet50Layers(ResNetLayers):
+
+    """A pre-trained CNN model with 50 layers provided by MSRA [1].
```

Comment:
OK, how about this?

> ResNet50 has 58% and 43% fewer parameters compared to ResNet101 and ResNet152, while the top-5 classification accuracy drop on ImageNet dataset are 0.7% and 1.1% from ResNet101 and ResNet152, respectively. Therefore, ResNet50 may have the best balance between the accuracy and the model size. It would be basically just enough for many cases, but some advanced models for object detection or semantic segmentation use deeper ones as their building blocks, so they are here for making reproduction work easier.